x/evm/keeper/evm.go

```diff
@@ -34,6 +34,7 @@ func (k *Keeper) HandleInternalEVMCall(ctx sdk.Context, req *types.MsgInternalEV
 	if err != nil {
 		return nil, err
 	}
+	k.GetEVMAddressOrDefault(ctx, senderAddr)
```

Comment:
why do we duplicate this effort with the below line?

Reply:
I think this is a mistake--will remove